Makefile

```diff
@@ -16,6 +16,8 @@ DOCKER_BUILD=docker build --cache-from
 
 DEVELOPMENT_DB_URI = postgresql://postgres:@api-database:5432/postgres?sslmode=disable
 
+GOOGLEAPIS_DIR=./proto/third_party/googleapis
```

Comment:
do we have to include the source code of third party library into our repo?  can we do a dynamic check out and build?

Reply:
If we must include 3rd party source code, what's the version? And what is the source url?

Reply:
The license, version and source url were in the same folder as the proto definitions. But I removed the 3rd party code as we actually don't depend on them after testing it again.